.travis.yml

```diff
@@ -36,8 +36,6 @@ cache:
   apt: true
   directories:
   - node_modules
-before_script:
-- npm install -g gemini@5.2.0 html-reporter@2.0.1
```

Comment:
Did we change something in travis builds? How come this is not needed anymore??

Reply:
I moved `gemini` and `html-reporter` to the `devDependencies` in package.json, so it is one less thing to do when setting up Clarity. The scripts were updated to run against the version installed in node_modules instead of globally, which we should really avoid using global packages anyways.